elastic/kibana
# [Stack Monitoring] add default sort props in react version

Resolves https://github.com/elastic/kibana/issues/114292

Makes tables default to Name, Ascending

### Test
- clear your local storage by opening up dev tools and entering `localStorage.clear()` in the console
- navigate to the elasticsearch nodes listing page and view the table.  it should have an arrow pointing up over the name column 
![Screen Shot 2021-10-07 at 10 28 54 AM](https://user-images.githubusercontent.com/1676003/136407605-b57b3917-4231-40b6-8178-0ecef56e54f6.png)
- click on another column, refresh the page, ensure the last sorted settings are still in place
